sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -7913,6 +7912,173 @@ __ESIMD_API
       offsets + glob_offset + __ESIMD_DNS::localAccessorToOffset(acc), mask);
 }
 
+/// Variant of scatter that uses local accessor as a parameter
+/// template <typename T, int N, int VS = 1, typename AccessorT,
+/// 	  typename PropertyListT = empty_properties_t>
+/// void scatter(AccessorT acc, simd<uint32_t, N / VS> byte_offsets, simd<T, N>
+/// vals, 	simd_mask<N / VS> mask, PropertyListT props = {}); //
+/// (lacc-sc-1)
```

Comment:
nit: It is something with indentations here and other places too (especially for `vals` operand): 
```suggestion
/// template <typename T, int N, int VS = 1, typename AccessorT,
/// 	      typename PropertyListT = empty_properties_t>
/// void scatter(AccessorT acc,
///              simd<uint32_t, N / VS> byte_offsets, simd<T, N> vals,
///              simd_mask<N / VS> mask,
///              PropertyListT props = {});                       // (lacc-sc-1)
```